wwwroot/data/regionids/region_map-FID_LGA_2013_AUST_STE_NAME11.json

```diff
@@ -0,0 +1 @@
estern Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Australian Capital Territory","Australian Capital Territory","Other Territories","Other Territories"]}
```

Comment:
Also is that an extraneous carriage return in there, or is that just github's rendering?